src/ir/var.rs

```diff
@@ -220,7 +220,7 @@ impl ClangSubItemParser for Var {
                     handle_function_macro(&cursor, &tokens, callbacks)?;
                 }
 
-                let value = parse_macro(ctx, &tokens);
+                let value = parse_macro(ctx, &cursor.spelling(), &tokens);
```

Comment:
It seems odd to pass the name via `cursor.spelling()` if we're not using that as the real name (in fact `modify_macro` could actually change the macro name...). But anyway.

Reply:
Yeah! Probably the interface would be better if we pass the name as a String.

For the tokens, we could pass the slice that omits the macro name; that would work for my use case. However my line of thinking in passing the mutable reference to the tokens is that as part of the  process we could want to change the name. For instance, another reasonable approach for my use case is to transform
 ```
#define FLAG_BITS 8:10
```
into
```
#define FLAG_HI_BIT 10
#define FLAG_LO_BIT 8
```